lib/agent0/agent0/hyperdrive/policies/lpandarb.py

```diff
@@ -0,0 +1,642 @@
+        low_fixed_rate_thresh: FixedPoint = FixedPoint("0.02")
+        rate_slippage: FixedPoint = FixedPoint("0.01")
+
+        @property
+        def arb_portion(self) -> FixedPoint:
+            """The portion of capital assigned to arbitrage."""
+            return FixedPoint(1) - self.lp_portion
+
+    def __init__(
+        self,
+        budget: FixedPoint,
+        rng: NumpyGenerator | None = None,
+        slippage_tolerance: FixedPoint | None = None,
+        policy_config: Config | None = None,
+    ):
+        """Initialize the bot.
+
+        Arguments
+        ---------
+        budget: FixedPoint
+            The budget of this policy
+        rng: NumpyGenerator | None
+            Random number generator
+        slippage_tolerance: FixedPoint | None
+            Slippage tolerance of trades
+        policy_config: Config | None
+            The custom arguments for this policy
+        """
+        # Defaults
+        if policy_config is None:
+            policy_config = self.Config()
+        self.policy_config = policy_config
+        self.arb_amount = self.policy_config.arb_portion * budget
+        self.lp_amount = (self.policy_config.lp_portion) * budget
+        self.minimum_trade_amount = FixedPoint("10")
+        # calculate these once
+        self.fp0 = FixedPoint(0)
+        self.fp1 = FixedPoint(1)
+        self.fp365 = FixedPoint(365)
+
+        super().__init__(budget, rng, slippage_tolerance)
+
+    def action(
+        self, interface: HyperdriveInterface, wallet: HyperdriveWallet
+    ) -> tuple[list[Trade[HyperdriveMarketAction]], bool]:
+        """Specify actions.
+
+        Arguments
+        ---------
+        interface : HyperdriveInterface
+            Interface for the market on which this agent will be executing trades (MarketActions)
+        wallet : HyperdriveWallet
+            agent's wallet
+
+        Returns
+        -------
+        tuple[list[MarketAction], bool]
+            A tuple where the first element is a list of actions,
+            and the second element defines if the agent is done trading
+        """
+
+        action_list = []
+
+        # Initial conditions, open LP position
+        if wallet.lp_tokens == FixedPoint(0):
+            # Add liquidity
+            action_list.append(
+                Trade(
+                    market_type=MarketType.HYPERDRIVE,
+                    market_action=HyperdriveMarketAction(
+                        action_type=HyperdriveActionType.ADD_LIQUIDITY,
+                        trade_amount=self.lp_amount,
+                        wallet=wallet,
+                        min_apr=interface.fixed_rate - self.policy_config.rate_slippage,
+                        max_apr=interface.fixed_rate + self.policy_config.rate_slippage,
+                    ),
+                )
+            )
+
+        # arbitrage from here on out
+        high_fixed_rate_detected = interface.fixed_rate >= self.policy_config.high_fixed_rate_thresh
```

Comment:
based on your description I believe these should be `>` and `<`, not `>=` and `<=`

Reply:
don't think it matters, since hitting an exact dollar figure of interest rates is probably never going to happen. leaving it since I'm changing description to: "higher than the variable rate by `high_fixed_rate_thresh`".